src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -3571,7 +3581,41 @@ private void EmitModuleVersionIdStore(BoundModuleVersionId node)
 
         private void EmitModuleVersionIdToken(BoundModuleVersionId node)
         {
-            _builder.EmitToken(_module.GetModuleVersionId(_module.Translate(node.Type, node.Syntax, _diagnostics.DiagnosticBag), node.Syntax, _diagnostics.DiagnosticBag), node.Syntax, _diagnostics.DiagnosticBag);
+            _builder.EmitToken(
+                _module.GetModuleVersionId(_module.Translate(node.Type, node.Syntax, _diagnostics.DiagnosticBag), node.Syntax, _diagnostics.DiagnosticBag),
+                node.Syntax,
+                _diagnostics.DiagnosticBag);
+        }
+
+        private void EmitThrowIfModuleCancellationRequested(SyntaxNode syntax)
```

Comment:
Do we ensure the helpers are available when we create the node?

Reply:
Yes, the instrumenter that uses this checks the availability:

https://github.com/dotnet/roslyn/pull/71896/files#diff-61e54648a0c8e35adbcd040f5e0bff216826c00c393396aaa72e418682be5cbdR48-R49